### PR TITLE
Run nightly tests on GitHub Actions

### PR DESCRIPTION
**Pull request by samsymons**
Task/Issue URL: https://app.asana.com/0/414235014887631/1203373812154613/f
Tech Design URL:
CC:

**Description**:

This PR adds a new workflow file that runs the Fingerprinting and Atb test suites every night.

**Steps to test this PR**:
1. GitHub Actions will only run workflows on a schedule if they are present on the default branch. I had them run on every commit previously, which you can see [here](https://github.com/duckduckgo/iOS/actions/workflows/nightly.yml). Otherwise, check that the cron syntax looks good.

**Copy Testing**:

* [ ] Use of correct apostrophes in new copy, ie `’` rather than `'`

**Orientation Testing**:

* [ ] Portrait
* [ ] Landscape

**Device Testing**:

* [ ] iPhone SE (1st Gen)
* [ ] iPhone 8
* [ ] iPhone X
* [ ] iPhone 14 Pro
* [ ] iPad

**OS Testing**:

* [ ] iOS 14
* [ ] iOS 15
* [ ] iOS 16

**Theme Testing**:

* [ ] Light theme
* [ ] Dark theme

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**

### CODE REVIEW

File: FingerprintingUITests/FingerprintUITest.swift [hunk 1]
```diff
@@ -120,7 +120,7 @@ class FingerprintUITest: XCTestCase {
         
         // Clear all tabs and data
         app.toolbars["Toolbar"].buttons["Fire"].tap()
-        app.sheets.scrollViews.otherElements.buttons["Close Tabs and Clear Data"].tap()
+        app.buttons["Close Tabs and Clear Data"].tap()
```
Comment (samsymons) [inline FingerprintingUITests/FingerprintUITest.swift:123]: This was a legitimate failure for me locally.

File: .github/workflows/nightly.yml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    steps:
+    - name: Check out the code
+      uses: actions/checkout@v3
+      with:
+        submodules: recursive
+
+    - name: Build and test
+      run: |
+        set -o pipefail && xcodebuild test \
+          -scheme "AtbUITests" \
```
Comment (samsymons) [inline .github/workflows/nightly.yml:25]: This is the only difference between these two jobs, so I'd be happy to move these to a matrix if necessary.

**Comment by samsymons** [comment]
One note on this is that it'd be nice to get notified when these fail. I'll look into that before this gets merged.

**Review by brindy — APPROVED** [review]
LGTM

**Pull request merged**